### ISSUE TITLE
standalone.js: not ES5 compatible anymore since 1.15.0 -> Unexpected token: keyword (const)

### ISSUE THREAD

**Issue by christophercr** [prettier/prettier#5461]
**Environments:**
- Prettier Version: 1.15.x
- Running Prettier via: Webpack, UglifyJS
- Runtime: Node.js v6, Node.js v8
- Operating System: ALL

**Steps to reproduce:**

Build your application with Webpack and UglifyJS targeting ES5.

**Expected behavior:**
The applications should build correctly with ES5 without using any parser to compile external libraries (i.e. Prettier).

**Actual behavior:**
The build fails due to:

```
ERROR in vendor.b5330e7b0af9bd05ebf3.chunk.js from UglifyJs
Unexpected token: keyword (const) [./node_modules/prettier/standalone.js:18962,0][vendor.b5330e7b0af9bd05ebf3.chunk.js:45354,0]
```

The issue was introduced in this commit: https://github.com/prettier/prettier/commit/91206891cd7b6209f1e7213f8cb43f64e0eec857#diff-863f7f2568e0a7b62f866fcadb68d31e  as part of this PR https://github.com/prettier/prettier/pull/5040 for Prettier 1.15.x.

Basically, this destructuring assignment:

```javascript
const {
   cjkPattern,
   kPattern,
   punctuationPattern
 } = require("./constants.evaluate");
```

is not correctly parsed to ES5 when building/releasing the Prettier package. In the distribution package the `standalone.js` file is like this:

```javascript
// "const" is not ES5, but ES6 standard
const json$9 = {"cjkPattern":"[...]","kPattern":"[...]","punctuationPattern":"[...]"};

var cjkPattern = json$9.cjkPattern;
var kPattern = json$9.kPattern;
var punctuationPattern$1 = json$9.punctuationPattern;
```

**Comment by lydell** [comment]
Note to maintainers: We should probably add a test verifying that the file is ES5.

**Comment by j-f1** [comment]
We could try running some basic tests on Node v0.10 on the standalone bundle, since https://node.green indicates it supports almost no ES6 features.

**Comment by lydell** [comment]
I think it's easier to run ESLint in ES5 mode on it.

**Comment by SimenB** [comment]
>I think it's easier to run ESLint in ES5 mode on it.

That's what Jest does. https://github.com/facebook/jest/blob/7a047f4826aa3bd24ac3399caa45a8b88033a393/package.json#L91
